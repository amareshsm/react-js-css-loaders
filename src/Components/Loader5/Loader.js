import React from 'react'
import styles from './style.css'
const Loader5 = (props) => (
  <>
    <style>{`${styles}`}</style>
    <div className="loaders">
      <div className="loader-div five">
        <div className="loader5"></div>
      </div>
    </div>
  </>
)

export default Loader5
