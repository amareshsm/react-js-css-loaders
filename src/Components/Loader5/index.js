export { default as Loader5 } from './Loader'
