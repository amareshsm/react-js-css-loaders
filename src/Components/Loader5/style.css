.loaders {
    display: flex;
    justify-content: center;
    box-sizing: border-box;
}

.loader5,
.loader5::before,
.loader5::after {
    width: 20px;
    height: 20px;
    border-radius: 50%;
    animation: loader5 1.5s ease-in-out infinite;
}

.loader5 {
    transform: translateZ(0);
    animation-delay: -0.16s;
    color: black;
}

.loader5::before,
.loader5::after {
    position: absolute;
    top: 0;
    content: "";
}

.loader5::before {
    left: -36px;
    animation-delay: -0.32s;
}
.loader5::after {
    left: 36px;
}

@keyframes loader5 {
    0%,
    80%,
    100% {
        box-shadow: 0 40px 0 -20px;
    }
    40% {
        box-shadow: 0 40px 0 0;
    }
}
