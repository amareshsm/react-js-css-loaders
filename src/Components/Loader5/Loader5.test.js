import React from 'react'
import renderer from 'react-test-renderer'
import { Loader5 } from '../Loader5'
it('renders correctly', () => {
  const tree = renderer.create(<Loader5 />).toJSON()
  expect(tree).toMatchSnapshot()
})
