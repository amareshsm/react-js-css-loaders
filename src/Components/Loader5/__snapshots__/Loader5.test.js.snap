// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <style>
    [object Object]
  </style>,
  <div
    className="loaders"
  >
    <div
      className="loader-div five"
    >
      <div
        className="loader5"
      />
    </div>
  </div>,
]
`;
